### DISCUSSION TITLE
Feature Request: Display and Select Commit to Deploy in Github Actions

### DISCUSSION THREAD

**Discussion by DieterVanhove** [community/community#52049] [Actions]
### Select Topic Area

Product Feedback

### Body

I think it would be really helpful to see which commit I'm deploying when using Github Actions. Currently, the only way to do this is by inserting the commit SHA manually. However, in many cases, I simply want to deploy the latest commit and verify that it is indeed the most recent one, without having to double-check the SHA. It would save a lot of time and effort if there was a more straightforward way to accomplish this.

One potential solution would be to add a dropdown menu that displays the available commits, allowing the user to easily select the one they want to deploy. This would be particularly useful in situations where there are multiple recent commits, or when deploying code across multiple branches. Of course, this is just a suggestion and may not be the right solution for everyone, but I think it would be a valuable addition to Github Actions.

<img width="1725" alt="image" src="https://user-images.githubusercontent.com/33942009/230047017-bdebda57-cc36-4380-832c-0ff8eeec4d4e.png">

**Comment by parvez-nawaz** [comment]
I have the same issue.
